.github/workflows/test.yml

```
@@ -55,15 +53,12 @@ jobs:
               nextcloud-versions: ['master']
               db: ['sqlite']
               include:
-                - php-versions: 8.1
-                  nextcloud-versions: stable29
-                  db: 'sqlite'
-                - php-versions: 8.2
-                  nextcloud-versions: stable29
-                  db: 'sqlite'
-                - php-versions: 8.3
+                - php-versions: '8.0'
```

```suggestion
                - php-versions: '8.1'
```

30 supports 8.1, not 8.0